.github/actions/linter_tests/action.yaml

```diff
@@ -44,11 +44,15 @@ runs:
           Linux)
             sudo apt-get update
             sudo apt-get -y install libperl-critic-perl perltidy zlib1g-dev
+            DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC sudo apt-get -y install tzdata
```

Comment:
Why do we need the environment vars here? Can you leave a comment if they're necessary?

Reply:
left a comment